### PR TITLE
created styles folder, added colors, text styles. 

### PR DESCRIPTION
**Pull request by vanesng**
+ +created button components folder 
+ +created teachers screens folder

### CODE REVIEW

File: screens/teachers/TeacherProfileEditScreen.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Button, Text, View } from 'react-native';
+import { styles } from '../../config/styles';
```
Comment (loischang) [inline screens/teachers/TeacherProfileEditScreen.js]: import from styles

File: screens/teachers/TeacherProfileScreen.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Button, Text, View } from 'react-native';
+import { styles } from '../../config/styles';
```
Comment (loischang) [inline screens/teachers/TeacherProfileScreen.js]: import from styles

File: styles/textStyles.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  linkText: {
+    color: 'primaryYellow',
+    fontSize: '20'
+  },
+  
+
+});
+
+export { textStyles };
```
Comment (loischang) [inline styles/textStyles.js]: nit: add new line at end of file

File: styles/colors.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  textLight: '#FFF', 
+
+  backgroundGrey: '#F5F5F6', 
+  dividerGrey: '#E0E0E0', 
+  iconDark: 'rgba(33, 33, 33, .85)',
+  iconLight: 'rgba(255, 255, 255, .85)',
+
+});
+
+export { colors };
```
Comment (loischang) [inline styles/colors.js]: nit: add new line at end of file

**Review by loischang — CHANGES_REQUESTED** [review]

**Review by loischang — APPROVED** [review]

**Comment by loischang** [comment]
LGTM  :D

File: screens/teachers/TeacherProfileEditScreen.js (hunk 1 above)
Comment (vivekp-toast) [reply screens/teachers/TeacherProfileEditScreen.js]: Fixed in 2d974a8. The header bullet (line 8) and the revision note now both read TC-101–140.

**Pull request merged**